index.html

```diff
@@ -31,8 +31,7 @@
     <div class="wrapperSections">
         <section class='section' id='newsletter'>
             <div class="tablet">
-                <span id='joinSpan'>Join our</span>
-                <span id='newsSpan'>newsletter</span>
+                <img src="./assets/newsletter.svg" alt="Join our newsletter">
```

Comment:
Should we add something like "letters spelling Join our newsletter" to be more descriptive? What is the standard for images with text?